quarkchain/cluster/jsonrpc.py

```diff
@@ -865,15 +871,16 @@ def get_data_default(key, decoder, default=None):
     @decode_arg("header_hash", hash_decoder)
     @decode_arg("nonce", quantity_decoder)
     @decode_arg("mixhash", hash_decoder)
-    async def submitWork(self, full_shard_key, header_hash, nonce, mixhash):
+    @decode_arg("signature", signature_decoder)
+    async def submitWork(self, full_shard_key, header_hash, nonce, mixhash, signature="0x" + bytes(65).hex()):
```

Comment:
signature=bytes(65)?

Reply:
I think it is a string. After the @decode_arg("signature", signature_decoder), it is converted to bytes(65).